doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -58,113 +59,109 @@ sdk\<service name>\<package name>\CHANGELOG.md
 sdk\<service name>\<package name>\<package name>.sln
 ```
 
-- `<service name>` - Should be the short name for the azure service. e.g. deviceupdate
+- `<service name>` - Should be the short name for the azure service. e.g. deviceupdate. It is usually your service name in REST API specifications. For instance, if the API spec is under `specification/storage`, the service would normally be `storage`.
 - `<package name>` -  Should be the name of the shipping package, or an abbreviation that distinguishes the given shipping artifact for the given service. It will be `Azure.<group>.<service>`, e.g. Azure.IoT.DeviceUpdate
 
-### Use Cadl as Input
+### Use Cadl as Input API spec
 
 #### Create Cadl project
   
   You can download existing cadl project from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo or you can follow following steps to create a new cadl project.
 
   ***Initialize Cadl Project***
 
-  Follow [Cadl Getting Start](https://github.com/microsoft/cadl/#using-node--npm) to initialize your Cadl project.
+  Follow [Cadl Getting Start](https://github.com/microsoft/typespec/#using-node--npm) to initialize your Cadl project.
 
   Make sure `npx cadl compile .` runs correctly.
```

Comment:
This is misleading as a customer will never need to run this if they use `dotnet build /t:GenerateCode`

Reply:
same as above

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/34058#discussion_r1114822746